docs/dev-guide/index.md

```diff
@@ -188,8 +188,10 @@ const question = await queryRow(sql.select_question, { question_id: 45 }, Questi
 
 - To be able to use the stored procedures from the `psql` command line it is necessary to get the most recent schema name using `\dn` and set the `search_path` to use this _quoted_ schema name and the `public` schema:
 
-  ```
-  set search_path to "server_2021-07-07T20:25:04.779Z_T75V6Y",public;
+  ```sql
+  set
+    search_path to "server_2021-07-07T20:25:04.779Z_T75V6Y",
+    public;
```

Comment:
Can we make Prettier ignore this, for the very dumb reason that I grep for `set search_path` whenever I want to find this line?

Reply:
Resolved in 766978d32.